compose/desktop/desktop/samples-material3/build.gradle

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
Is this code taken from androidx? If it is not, let's convert it to .kts, since it is a default approach now in Gradle

Reply:
Let's not fight with it then, since it can be exhausting. For me it is okey to merge